app/controllers/api/v4/questionnaires_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V4::QuestionnairesController < Api::V4::SyncController
+  def sync_to_user
+    __sync_to_user__("questionnaires")
+  end
+
+  def transform_to_response(questionnaire)
+    Api::V4::QuestionnaireTransformer.to_response(questionnaire)
+  end
+
+  def current_facility_records
+    []
+  end
+
+  def other_facility_records
```

Comment:
Why not just override `records_to_sync`?

Reply:
In `response_process_token`, we're following the convention followed in other Sync controllers.
```ruby
      other_facilities_processed_since: processed_until(other_facility_records) || other_facilities_processed_since,
```
So, overriding `other_facility_records` makes more sense.

Reply:
Since you're overriding `response_process_token` you could define it as `processed_until(records_to_sync)`.

I guess my gripe with this naming is that the questionnaires are for the current facility, not other facilities. I see that we have done similar naming in facilities sync as well, but there I think it works because we are syncing other facilities anyways. 

This is not a blocker, either ways.